Rules/CommonScripts/ChatCommands/PlayerCommands.as

```diff
@@ -207,3 +209,27 @@ class HealCommand : ChatCommand
 		}
 	}
 }
+
+class SuicideCommand : ChatCommand
+{
+	SuicideCommand()
+	{
+		super("suicide", "Kill yourself");
+		AddAlias("die");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (isServer())
+		{
+			CBlob@ blob = player.getBlob();
+			if (blob is null)
+			{
+				server_AddToChat(getTranslatedString("You cannot kill yourself while dead or spectating"), ConsoleColour::ERROR, player);
+				return;
+			}
+
+			blob.server_Hit(blob, blob.getPosition(), blob.getVelocity(), 10.0f, Hitters::suicide);
```

Comment:
On second thought, would it not be better to use blob.server_SetTimeToDie, at least then death will always happen

Reply:
Doesn't SetTimeToDie despawn the blob altogether? We just want to kill it on the battle field.

Reply:
It looks like for SetTimeToDie to work, you have to set a positive value above 0.
However, we don't want to kill the blob in the future. We want to kill it now.
Are there cases where server_Hit used like this will not result in the player's death?

Reply:
only in mods where players have crazy amounts of health. You could probably just set the damage output to be double that of the blob's initial health or something. 10.0f should be plenty though so its whatever